### PR TITLE
refactor(Config Schema): Use strict: true for schema validation

### PR DESCRIPTION
**Pull request by pgrzesik**
Ensure use of strict: true for AJV + fix issues with existing schema

### CODE REVIEW

File: lib/classes/ConfigSchemaHandler/resolveAjvValidate.js [hunk 1]
```diff
@@ -36,7 +36,6 @@ const getValidate = async (schema) => {
       allErrors: true,
       coerceTypes: 'array',
       verbose: true,
-      strict: false,
```
Comment (medikoo) [inline lib/classes/ConfigSchemaHandler/resolveAjvValidate.js:39]: This one could be added to AJV related commit

File: lib/configSchema.js [hunk 2]
```diff
@@ -87,6 +87,7 @@ const schema = {
       additionalProperties: false,
     },
     params: {
+      type: 'object',
```
Comment (medikoo) [inline lib/configSchema.js:90]: This actually should be added to `params` related commit

**Review by medikoo — APPROVED** [review]
@pgrzesik Looks great. I however propose to include those changes in already existing commits

**Comment by pgrzesik** [comment]
@medikoo pushed the changes to suggested commits on v3 branch

**Pull request closed without merging**